insights/combiners/identity_domain.py

```diff
@@ -0,0 +1,291 @@
+"""
+Identity Domain - Combiner for domain enrollment
+================================================
+
+The combiner detects enrollment into identity domains such as IPA,
+Active Directory, generic Kerberos realm, and generic LDAP. It parses
+domains and realms from SSSD, KRB5, IPA, and Samba configuration.
+
+Supported domain types
+----------------------
+
+* IPA (RHEL IdM, FreeIPA)
+* Active Directory (SSSD)
+* Active Directory (Samba winbind)
+* generic LDAP domain (SSSD)
+* generic LDAP domain with Kerberos authentication (SSSD)
+* generic Kerberos realm (from ``krb5.conf``)
+
+The combiner cannot detect generic Kerberos realms that solely rely upon
+DNS realm lookup (``dns_lookup_realm``).
+
+Examples::
+
+    DomainInfo(
+        name="ipa.test",
+        domain_type="IPA",
+        server_software="IPA",
+        client_software="SSSD",
+        domain="ipa.test",
+        realm="IPA.TEST",
+        workgroup=None,
+        ipa_mode="client",
+        is_default=True,
```

Comment:
IIUC, you might miss this attribute in the definition of the `DomainInfo`, or this item should be removed from the Examples here.

Reply:
Thanks! I missed the lines when I simplified the object.